src/prpy/tsr/tsr.py

```diff
@@ -51,15 +51,20 @@ def __init__(self, T0_w=None, Tw_e=None, Bw=None,
         pibound = (abs(Bw[3:6, :]) < numpy.pi + EPSILON)
         if pibound.any() is False:
             raise(ValueError('Rotations must be [-pi, pi]', pibound))
-        # We will now deliberately exceed the bound to maintain both:
-        # 1. Bw[i,1] > Bw[i,0] which is necessary for LBFGS-B
-        # 2. signed rotations, necessary for expressiveness
-        for rot_idx in range(3, 6):
-            if Bw[rot_idx, 0] > Bw[rot_idx, 1] + EPSILON:
-                Bw[rot_idx, 1] += 2*pi
+        
         self.T0_w = T0_w
         self.Tw_e = Tw_e
         self.Bw = Bw
+        
+        # We will now create a continuous version of the bound to maintain:
+        # 1. Bw[i,1] > Bw[i,0] which is necessary for LBFGS-B
+        # 2. signed rotations, necessary for expressiveness
```

Comment:
I think I see what's going on here, but I'm confused by this comment. This code converts a signed rotation into a rotation that exceeds the [ -pi, pi ] range, so it does *not* "maintain signed rotations".